video/out/wayland_common.c

```diff
@@ -3452,6 +3452,7 @@ static void set_color_management(struct vo_wayland_state *wl)
         return;
 
     struct pl_color_space color = wl->target_params.color;
+    struct wl_event_queue *cm_queue = NULL;
```

Comment:
Nit but why declare this NULL up here and not just when it gets created?

Reply:
Leftover from when I was trying something else, will change it